python-package/brainflow/data_filter.py

```diff
@@ -383,6 +384,7 @@ def perform_lowpass(cls, data: NDArray[Float64], sampling_rate: int, cutoff: flo
         :param ripple: ripple value for Chebyshev filter
         :type ripple: float
         """
+        check_memory_layout_row_major(data)
```

Comment:
note that this is the same conceptual kind of check as https://github.com/brainflow-dev/brainflow/pull/253/files#diff-d5c3bcf84e585e262092315dad35c121847283e2084c58a0281297dde4cfcbf0R392 and applies to all the same places, so the code could be merged